code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -393,3 +393,37 @@ Contains:
 		torn = TRUE
 		playsound(loc, 'sound/weapons/slashmiss.ogg', 50, 1)
 		playsound(loc, 'sound/effects/refill.ogg', 50, 1)
+

```

Comment:
You must move this content into the location that Karmic said. Or provide a reason as to why you are not moving it.